components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/AnalyticsParams.kt

```diff
@@ -26,14 +26,14 @@ data class AnalyticsParams(
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 enum class AnalyticsParamsLevel(val priority: Int) {
-    NONE(1),
+    INITIAL(1),
     ALL(2),
 }
 
 private fun getLevel(analyticsConfiguration: AnalyticsConfiguration?): AnalyticsParamsLevel {
```

Comment:
In case the `analyticsConfiguration` is not passed, we want to default the value to `ALL`.
For the when statement, it is already exhaustive,  since `null`  case is different than `else` case. If we add a new value, we will have to implement it as well.